config/hardware/probes/tap_probe.cfg

```diff
@@ -18,6 +18,23 @@ samples_result: median
 sample_retract_dist: 1.0
 samples_tolerance: 0.005
 samples_tolerance_retries: 3
+# Add safety feature before probing bed
+activate_gcode:
+    {% set PROBE_TEMP = printer["gcode_macro _USER_VARIABLES"].tap_max_probing_temp|float %}
+    {% set MAX_TEMP = PROBE_TEMP + 5 %}
+    {% set ACTUAL_TEMP = printer.extruder.temperature %}
+    {% set TARGET_TEMP = printer.extruder.target %}
+
+    {% if TARGET_TEMP > PROBE_TEMP %}
+        { action_respond_info('Extruder temperature target of %.1fC is too high, lowering to %.1fC' % (TARGET_TEMP, PROBE_TEMP)) }
+        M109 S{ PROBE_TEMP }
+    {% else %}
+        # Temperature target is already low enough, but nozzle may still be too hot.
+        {% if ACTUAL_TEMP > MAX_TEMP %}
+            { action_respond_info('Extruder temperature %.1fC is still too high, waiting until below %.1fC' % (ACTUAL_TEMP, MAX_TEMP)) }
+            TEMPERATURE_WAIT SENSOR=extruder MAXIMUM={ MAX_TEMP }
+        {% endif %}
+    {% endif %}
```

Comment:
This can be removed as already managed by https://github.com/Frix-x/klippain/blob/main/macros/base/probing/generic_probe.cfg

Reply:
Removed this code part.